plugins/openshift-image-registry/app-config.janus-idp.yaml

```diff
@@ -0,0 +1,20 @@
+proxy:
+  endpoints:
+    '/openshift-image-registry/api':
+    target: <URL where k8s control plane for OpenShift cluster is running>
```

Comment:
Same remark as above about required env variables to make this config work by default.